src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -148,6 +146,7 @@ import {
 } from '../../../services/conversationsService.js'
 import { EventBus } from '../../../services/EventBus.js'
 import { addParticipant } from '../../../services/participantsService.js'
+import { generateFullConversationLink } from '../../../services/urlService.js'
```

Comment:
'generateFullConversationLink' is defined but never used

Reply:
🙈